exhibit_titles: ['Πάπυρος του Δερβενίου', 'Χρυσός των Μακεδόνων', 'Γεωμετρία']
exhibit_codes: ['a', 'b', 'c']
